### PR TITLE
Added a possibility to sort and renumber all motions in a category (closes #2259)

### LINKED ISSUE #2259: New category sort view
Add a new view to sort all motions of a category via drag&drop (similar to agenda sort view).

A button "numbering" should renumber the identifer of each (sorted) motion, e.g. B 1, B 2, ...
You can use the code snippet from my test branch to see the right post request url: https://github.com/OpenSlides/OpenSlides/compare/master...emanuelschuetze:NumberCategories2#diff-8ace1613d1310b7a1a40a1676da3cb6cR1027

### PR DESCRIPTION
**Pull request by FinnStutzenstein**
In the category list view is a new link "sort" to reorder the motions with drag and drop.
What should be displayed, if there is no motion or only one motion in the category?

### CODE REVIEW

File: openslides/motions/static/js/motions/site.js [hunk 1]
```diff
@@ -1065,6 +1078,49 @@ angular.module('OpenSlidesApp.motions.site', ['OpenSlidesApp.motions'])
...
+    'category',
+    'Motion',
+    'motions',
+    function($scope, $stateParams, $http, MotionTree, Category, category, Motion, motions) {
+        Category.bindOne(category.id, $scope, 'category');
+        Motion.bindAll({}, $scope, 'motions');
+        $scope.filter = { category_id: category.id,
+                          orderBy: 'identifier' };
+
+        $scope.$watch(function () {
```
Comment (normanjaeckel) [inline openslides/motions/static/js/motions/site.js]: The indention of the following lines looks messy.

File: openslides/motions/static/js/motions/site.js [hunk 2]
```diff
@@ -1065,6 +1078,49 @@ angular.module('OpenSlidesApp.motions.site', ['OpenSlidesApp.motions'])
...
+              return Motion.lastModified();
+        }, function () {
+            $scope.items = MotionTree.getTree(Motion.filter($scope.filter));
+        });
+
+        $scope.alert = {};
+        // Numbers all motions in this category by the given order in $scope.items
+        $scope.numbering = function (category) {
```
Comment (normanjaeckel) [inline openslides/motions/static/js/motions/site.js]: The `category` is part of the scope so we do not have to give it as function argument.

**Comment by normanjaeckel** [comment]
Thanks for your work. Some remarks.

**Comment by emanuelschuetze** [comment]
Retested with fresh bower_components:
Works now for me.

**Pull request merged**